src/app/components/Common/Button/index.tsx

```diff
@@ -10,14 +10,16 @@ export interface IButtonProps {
   color?: Colors;
   outline?: boolean;
   radius?: Borders;
+  className?: string;
 }
 
 class Button extends React.Component<IButtonProps> {
   public static defaultProps = {
     size: "medium",
     color: "primary",
     outline: false,
-    radius: "large"
+    radius: "large",
+    className: null
```

Comment:
set this to "" because in the DOM it will read 
class="btn btn--blah" vs class="btn btn--blah null"

Reply:
good catch